Makefile

```diff
@@ -76,7 +76,7 @@ reviewable: generate docs manifests helm.generate helm.docs lint ## Ensure a PR
 	@go mod tidy
 	@cd e2e/ && go mod tidy
 
-check-diff: reviewable ## Ensure branch is clean.
+check-diff: reviewable helm.schema.update ## Ensure branch is clean.
```

Comment:
Put this after `helm.generate in `reviewable` target instead please. :)

Reply:
Done.

Reply:
Thanks!